### DISCUSSION TITLE
Cannot launch 2.10.2 in Ubuntu 22.04.4

### DISCUSSION THREAD

**Discussion by jedimaster2001** [artisan-roaster-scope/artisan#1497] [Q&A]
nghiat@nghiat-PORTEGE-Z935:~$ artisan
nghiat@nghiat-PORTEGE-Z935:~$ 

I am try to launch Artisan but it doesn't work. Nothing happened both from terminal and from App Launcher.

**Comment by MAKOMO** [comment]
The v2.10.2 Linux builds have issue on some Linux platforms. Could you try the current [continuous builds](https://github.com/artisan-roaster-scope/artisan/releases/tag/continuous) if that works any better for you and report back? Thanks!

**Reply by jedimaster2001** [reply]
Thank you very much for your suggestion. I am now able to launch with continuous build on xubuntu 23.10. Will test it out further.
